dashboard/views.py

```diff
@@ -208,7 +202,7 @@ def file_access_within_week(request):
     # time 100 to show the percentage
     output_df["total_count"] = output_df["total_count"] * 100
     # round all numbers to one decimal point
-    output_df = output_df.round(1)
+    output_df = output_df.round(2)
```

Comment:
This comment doesn't match the change here. Should the rounding precision be a constant?

Reply:
fixed. Defined constant as DECIMAL_ROUND_DIGIT = 1